android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -680,6 +682,20 @@ private void verifySubscription() {
                 });
     }
 
+    @Override
+    public boolean onOptionsItemSelected(int itemId, @Nullable Bundle menuItemData) {
+        if (itemId == R.id.new_tab_menu_id) {
+            LayoutManagerChrome layoutManager = (LayoutManagerChrome) BraveReflectionUtil.getField(
+                    ChromeTabbedActivity.class, "mLayoutManager", this);
```

Comment:
Could you please add tests checking that field still exists. Same applies to `mDataProvider`.